include/class.forms.php

```diff
@@ -4023,7 +4023,7 @@ function render($options=array()) {
         <span style="display:inline-block;width:100%">
         <textarea <?php echo $rows." ".$cols." ".$maxlength." ".$class
                 .' '.Format::array_implode('=', ' ', $attrs)
-                .' placeholder="'.$config['placeholder'].'"'; ?>
+                .' placeholder="'.$this->field->getLocal('placeholder').'"'; ?>
```

Comment:
This assumes the placeholder is translated -  use config placeholder as fallback.

```php
$this->field->getLocal('placeholder', $config['placeholder'])
```

Reply:
Well, `getLocal()` should return the default if no translation is available via [this line](https://github.com/osTicket/osTicket/blob/develop/include/class.forms.php#L1331):
```
return $T != $tag ? $T : ($default ?: $this->get($subtag));
```

In the above line, we do `$default ?: $this->get($subtag)` (where `$default` is the default and `$subtag` is `placeholder` in this case) if no translation is available which should return the default English translation.

Reply:
But I get what you mean, in order to not refetch then we can pass a default

Reply:
updated